bindings/nodejs/README.md

```diff
@@ -29,7 +29,23 @@ async function main() {
 main();
 ```
 
-## Test or Contributing
+## Services
+
+- [azblob](https://docs.rs/opendal/latest/opendal/services/struct.Azblob.html): [Azure Storage Blob](https://azure.microsoft.com/en-us/services/storage/blobs/) services.
```

Comment:
I think adding links to rust code doesn't look good to node.js developers.

Reply:
Consider adding many practical examples as demonstrations and replacing the links here. 
However, this is quite difficult for us to maintain synchronously in multiple languages. 
I suggest that we could maintain them together on the website later and provide examples between different languages at the same time.

Reply:
LGTM, let's remove this part for now.